src/Test/Utilities/Portable/CompilationVerifier.cs

```diff
@@ -180,7 +186,37 @@ public void VerifyTypeIL(string typeName, string expected)
                     Assert.True(found, "Could not find type named " + typeName);
                 }
             }
-            AssertEx.AssertEqualToleratingWhitespaceDifferences(expected, output.ToString(), escapeQuotes: false);
+
+            var outputString = output.ToString();
+            if (normalize)
+            {
+                expected = NormalizePlatformSpecificCode(expected);
+                outputString = NormalizePlatformSpecificCode(outputString);
+            }
+            AssertEx.AssertEqualToleratingWhitespaceDifferences(expected, outputString, escapeQuotes: false);
+        }
+
+        /// <summary>
+        /// Used by <see cref="NormalizePlatformSpecificCode(string)"/>
+        /// </summary>
+        private static readonly (string toReplace, string replaceWith)[] PlatformSpecificNormalizations =
```

Comment:
Our style guidelines say this should be named `s_platformSpecificNormalizations`